tests/bluetooth/bsim_bt/bsim_test_adv_resume/test_scripts/_env.sh

```diff
@@ -11,8 +11,8 @@ bsim_bin="${BSIM_OUT_PATH}/bin"
 verbosity_level=2
```

Comment:
I know this PR is not the one introducing this script, and as such this comment is a bit off topic, but 2 things about the script:
* It is very weird to have a script be executed, having it printing out to stdout commands and then have those commands be interpreted by the user script. You can just define any variables you want in a script and have the user simply do `source my_script.sh`. Meaning this script does not need the print_var function or its calls.
* Minor: This script aim seems to be to share the variable names and directory for the test executable. But you are also setting other variables (sim_id, verbosity) here used in only one place (which hides them, and makes the code more complex to understand). I wouldn't define here neither of the later 2.

Reply:
Yeah I find this script weird too, I think @alwa-nordic introduced it, he was using it for local development. I'll simplify it.